math/dec_test.go

```diff
@@ -860,8 +1216,8 @@ func TestToBigInt(t *testing.T) {
 		require.NoError(t, err)
 		b, err := a.BigInt()
 		if tc.isError == nil {
-			require.NoError(t, err, "test_%d", idx)
-			require.Equal(t, tc.out, b.String(), "test_%d", idx)
+			require.NoError(t, err, "test_/d", idx)
```

Comment:
please revert this and below. `%d` is a formatting instruction for decimals. 
The use of idx inside the asserts can be pulled up into the name when the test suite would start `t.Run(idx or name, func(` for each case.